src/components/illustrations/XPIcon.tsx

```diff
@@ -1,57 +1,64 @@
-import { useTheme } from '@mui/material';
-import React from 'react';
+import type { BoxProps } from '@mui/material';
+import { alpha, Box, styled, useTheme } from '@mui/material';
 
-// XPIcon
 interface XPIconProps {
+  color?: string;
+  bgColor?: string;
   size?: number;
+  variant?: 'primary' | 'secondary' | 'completed';
 }
 
-export const XPIcon = ({ size }: XPIconProps) => {
+export const XPIcon = ({
```

Comment:
XPIcon and IconXP seems to be the same, needs refactoring

Reply:
fair